src/ProgressBar.js

```diff
@@ -27,7 +27,7 @@ const ProgressBar = React.createClass({
   },
 
   getPercentage(now, min, max) {
-    return Math.ceil((now - min) / (max - min) * 100);
+    return Math.round(((now - min) / (max - min) * 100)*1000)/1000;
```

Comment:
Can you assign 100 and 1000 to a variable name to explain their purpose. This will ensure ease of maintainability in the future.